src/components/AddItems.js

```diff
@@ -91,8 +92,15 @@ const AddItemsToList = () => {
   return (
     <div>
       <div>
+        <header className="bg-green-400 w-full fixed text-center">
+          <h2 className="pt-6 pb-16 text-4xl font-thin text-gray-100">
+            Add Item to List
+          </h2>
+          <span className="text-white top-0 right-0 absolute sm:mt-4 sm:mr-4">
+            <HomeIcon />
```

Comment:
Also, the home icon, it doesnt lead to the home page, is it intentional or an omission. I suggest to the entire team that we take it out since we dont have a "home" button on the additems and itsmlist components

Reply:
Yeah, well, we were discussing if we will do a return function in our last meeting or not. I only did the styling on this one, the code should be done next week if we decide to go with it.

Reply:
I think it would be a good idea to implement the home functionality, if we have time! If not then we can always remove it.